README.md

```diff
@@ -102,9 +56,7 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
```

Comment:
- [ ]  @team kindly remove this line, its suppose to guide you when writing the readme but shouldn't be retained in the readme.